docs/source/new-project.rst

```diff
@@ -358,9 +357,9 @@ And the matching code:
 
 |
 
-.. testcode:: python
```

Comment:
Lets keep this doc-tested. I'll add changes to this PR to do this. Otherwise we'll never know when these examples start to break due to API changes.